os/drivers/audio/alc5658.c

```diff
@@ -912,7 +922,7 @@ static int alc5658_enqueuebuffer(FAR struct audio_lowerhalf_s *dev, FAR struct a
 
 	if (priv->inout) {		/* record */
 		ret = I2S_RECEIVE(priv->i2s, apb, alc5658_rxtxcallback, priv, CONFIG_ALC5658_I2S_TIMEOUT);
-	} else {					/* playback */
+	} else {				/* playback */
```

Comment:
Is this diff necessary and related with commit message?
It seems better to remove unnecessary diff chunck.

Reply:
agreed